src/core/lambda/io.rs

```diff
@@ -133,6 +137,18 @@ impl IO {
 
                     Ok(res.body)
                 }
+                IO::Js { name: method } => {
+                    let value = ctx.value().cloned().unwrap_or_default();
+                    let value = ctx
+                        .request_ctx
+                        .runtime
+                        .worker
+                        .call(method.clone(), value)
```

Comment:
Hot code should avoid clones as much as possible. Can you update the signature to pass the function name as an &str.

Reply:
difficult..